### ISSUE TITLE
RedisAI crashes on MODELRUN if INPUTS/OUTPUTS names are not present in the TF graph

### ISSUE THREAD

**Issue by hhsecond** [RedisAI/RedisAI#127]
If by mistake user sets the model with the wrong name in the `INPUTS` / `OUTPUTS` list, RedisAI crashes on MODELRUN without proper error message.

```python
client.modelset(
    'model', rai.Backend.tf, rai.Device.cpu, model, input=['WrongName'], output=['WrongName'])
```

From a UX standpoint, we should be raising an error while setting the model about the wrong data, IMO

**Comment by lantiga** [comment]
We test for omission but not for this, I'll fix right away

**Comment by lantiga** [comment]
Solved by #128
